tests/test_evpn_tunnel.py

```diff
@@ -0,0 +1,749 @@
+        print("Pass")
+        print("Testing VLAN 100 extension")
+        vxlan_obj.check_vlan_extension(dvs, '100', '7.7.7.7')
+        print("Pass")
+
+        create_evpn_remote_vni(dvs, 'Vlan101', '7.7.7.7', '1001')
+        create_evpn_remote_vni(dvs, 'Vlan102', '7.7.7.7', '1002')
+
+        print("Testing DIP tunnel not created again")
+        vxlan_obj.check_vxlan_dip_tunnel(dvs, tunnel_name, '6.6.6.6', '7.7.7.7')
+        print("Pass")
+
+        print("Testing VLAN 101 extension")
+        vxlan_obj.check_vlan_extension(dvs, '101', '7.7.7.7')
+        print("Pass")
+
+        print("Testing VLAN 102 extension")
+        vxlan_obj.check_vlan_extension(dvs, '102', '7.7.7.7')
+        print("Pass")
+
+        print("Testing another DIP tunnel to 8.8.8.8")
+        create_evpn_remote_vni(dvs, 'Vlan100', '8.8.8.8', '1000')
+        print("Testing DIP tunnel creation to 8.8.8.8")
+        vxlan_obj.check_vxlan_dip_tunnel(dvs, tunnel_name, '6.6.6.6', '8.8.8.8')
+        print("Pass")
+        print("Testing VLAN 100 extension to 8.8.8.8 and 7.7.7.7")
+        vxlan_obj.check_vlan_extension(dvs, '100', '8.8.8.8')
+        vxlan_obj.check_vlan_extension(dvs, '100', '7.7.7.7')
+        print("Pass")
+
+        print("Testing Vlan Extension removal")
+        remove_evpn_remote_vni(dvs, 'Vlan100', '7.7.7.7')
+        remove_evpn_remote_vni(dvs, 'Vlan101', '7.7.7.7')
+        vxlan_obj.check_vlan_extension_delete(dvs, '100', '7.7.7.7')
+        vxlan_obj.check_vlan_extension_delete(dvs, '101', '7.7.7.7')
+        print("Testing DIP tunnel not deleted")
+        vxlan_obj.check_vxlan_dip_tunnel(dvs, tunnel_name, '6.6.6.6', '7.7.7.7')
+        print("Pass")
+
+        print("Testing Last Vlan removal and DIP tunnel delete")
+        remove_evpn_remote_vni(dvs, 'Vlan102', '7.7.7.7')
+        vxlan_obj.check_vlan_extension_delete(dvs, '102', '7.7.7.7')
+        vxlan_obj.check_vxlan_dip_tunnel_delete(dvs, '7.7.7.7')
+        print("Pass")
+
+        print("Testing Last Vlan removal and DIP tunnel delete for 8.8.8.8")
+        remove_evpn_remote_vni(dvs, 'Vlan100', '8.8.8.8')
+        vxlan_obj.check_vlan_extension_delete(dvs, '100', '8.8.8.8')
+        vxlan_obj.check_vxlan_dip_tunnel_delete(dvs, '8.8.8.8')
+        print("Pass")
+
+        remove_vxlan_tunnel_map(dvs, tunnel_name, map_name, '1000', 'Vlan100')
+        remove_vxlan_tunnel_map(dvs, tunnel_name, map_name_1, '1001', 'Vlan101')
+        remove_vxlan_tunnel_map(dvs, tunnel_name, map_name_2, '1002', 'Vlan102')
+        vxlan_obj.check_vxlan_tunnel_map_entry_delete(dvs, tunnel_name, vlanlist, vnilist)
+
+        print("Testing SIP Tunnel Deletion")
+        remove_evpn_nvo(dvs, 'nvo1')
+        remove_vxlan_tunnel(dvs, tunnel_name)
+        vxlan_obj.check_vxlan_sip_tunnel_delete(dvs, tunnel_name)
+
+
+#    Test 3 - Create and Delete SIP Tunnel and Map entries
+#    @pytest.mark.skip(reason="Starting Vxlanmgr to be merged")
+    def test_p2mp_tunnel_with_dip(self, dvs, testlog):
```

Comment:
what is the use of testlog, not used anywhere? can all print's can be moved to testlog?

Reply:
test_vnet.py and test_vxlan_tunnel.py had references to testlog and hence inherited the same. 
Have removed it now

Reply:
`testlog` is a pytest fixture, it adds `test <blank> start` and `test <blank> end` to the syslogs to aid in debugging. It *should* probably just be auto-use so we don't have to explicitly invoke it everywhere, but that can be addressed in a different pr. 😄